Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -161,6 +162,19 @@ open class NavigationMapView: UIView {
         }
     }
     
+    var simulation: Bool = true
+    
+    /**
+     Specifies how the map displays the user’s current location, including the appearance and underlying implementation.
+     
+     By default, this property is set to `UserLocationStyle.courseView`.
+     */
+    public var puckType: UserLocationStyle = .courseView(configuration: UserPuckCourseView(frame: CGRect(origin: .zero, size: 75.0))) {
```

Comment:
Now that the enumeration has been renamed, let’s rename the property to `userLocationStyle` to match it.

Reply:
Thanks for the suggestion, I'll rename this property.